libs/sr/page-team/src/TeammateDisplay/Tabs/BonusStats.tsx

```diff
@@ -0,0 +1,216 @@
+}) {
+  const { database } = useDatabaseContext()
+  const {
+    loadout: { bonusStats },
+    loadoutId,
+  } = useContext(LoadoutContext)
+  const newTarget = (q: InitialStats) => {
+    const tag = newTag(q)
+    database.loadouts.set(loadoutId, (loadout) => {
+      loadout.bonusStats.push({
+        tag,
+        value: 0,
+      })
+    })
+  }
+  return (
+    <ModalWrapper open={open} onClose={onClose}>
+      <CardThemed>
+        <CardHeader title="Bonus Stats" />
+        <Divider />
+        <CardContent>
+          <Stack spacing={1}>
+            {bonusStats.map(({ tag, value }, i) => (
+              <BonusStatDisplay
+                key={JSON.stringify(tag) + i}
+                tag={tag}
+                value={value}
+                setValue={(value) => {
+                  database.loadouts.set(loadoutId, (loadout) => {
+                    loadout.bonusStats[i].value = value
+                  })
+                }}
+                onDelete={() => {
+                  database.loadouts.set(loadoutId, (loadout) => {
+                    loadout.bonusStats.splice(i, 1)
+                  })
+                }}
+                setTag={(tag) => {
+                  database.loadouts.set(loadoutId, (loadout) => {
+                    loadout.bonusStats[i].tag = tag
+                  })
+                }}
+              />
+            ))}
+            <InitialStatDropdown onSelect={newTarget} />
+          </Stack>
+        </CardContent>
+      </CardThemed>
+    </ModalWrapper>
+  )
+}
+function newTag(q: Tag['q']): Tag {
+  return {
+    et: 'own',
+    q,
+    qt: 'premod',
+    sheet: 'agg',
+  }
+}
+const initialStats: StatKey[] = [
+  'hp',
+  'hp_',
+  'def',
+  'def_',
+  'atk',
+  'atk_',
+  'spd',
+  'spd_',
+  'dmg_',
+  'enerRegen_',
+  'brEffect_',
+  'crit_',
+  'crit_dmg_',
+  'eff_',
+  'eff_res_',
+  'heal_',
+] as const
+type InitialStats = (typeof initialStats)[number]
+function InitialStatDropdown({
+  tag = {},
+  onSelect,
+}: {
+  tag?: Tag
+  onSelect: (key: (typeof initialStats)[number]) => void
+}) {
+  return (
+    <DropdownButton
+      title={(tag && tagFieldMap.subset(tag)[0]?.title) ?? 'Add Bonus Stat'}
+    >
+      {initialStats.map((statKey) => (
+        <MenuItem key={statKey} onClick={() => onSelect(statKey)}>
+          {statKey}
+        </MenuItem>
+      ))}
+    </DropdownButton>
+  )
+}
+
+function BonusStatDisplay({
+  tag,
+  setTag,
+  value,
+  setValue,
+  onDelete,
+}: {
+  tag: Tag
+  setTag: (tag: Tag) => void
+  value: number
+  setValue: (value: number) => void
+  onDelete: () => void
+}) {
+  // TODO: best way to infer percentage from tag?
+  const isPercent = (tag.name || tag.q)?.endsWith('_')
+  return (
+    <CardThemed bgt="light">
+      <CardContent
+        sx={{ display: 'flex', gap: 1, justifyContent: 'space-around' }}
+      >
+        <SqBadge sx={{ m: 'auto' }}>{tag.q}</SqBadge>
+        {tag.q === 'dmg_' && (
+          <ElementTypeDropdown
```

Comment:
technically this doesn't matter for HSR. I'm not aware of any character or mechanic that we would support in SRO that can do multi-element

Reply:
damageType1 and damageType2 would be better candidates, if we want to be technical about it

Reply:
I think you misunderstand. this differentiates between `dmg_` and `${element}_dmg_`, this is not a multi select.